File: src/main/java/noteedit/entity/Note.java
package com.gliese.noteedit.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Note {
    @TableId(type = IdType.AUTO)
    private Long id;
    private Long fileId;
    @TableField(value = "text")
    private String note;

    @Override
    public String toString() {
        return "Note{" +
                "id=" + id +
                ", fileId=" + fileId +
                ", note='" + (note.length() < 20 ? note.length() : (note.substring(0, 20) + "...")) + '\'' +
                '}';
    }
}

File: src/main/java/noteedit/entity/Note.kt
package noteedit.entity

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableField
import com.baomidou.mybatisplus.annotation.TableId

class Note(
    @TableId(type = IdType.AUTO)
    var id: Long = 0, // 主键
    var fileId: Long = 0, // 对应文件id
    @TableField("text")
    var note: String = "" // 笔记内容
) {
    override fun toString(): String {
        return "Note(id=$id, fileId=$fileId, note='${note.let {
            if (it.length > 10) it.substring(0, 10) + "..." else it
        }}')"
    }
}